protocols/ntree/ntree.go

```diff
@@ -74,6 +74,15 @@ func (p *Protocol) HandleSignRequest(msg structMessage) error {
 }
 
 func (p *Protocol) HandleSignReply(reply []structSignatureReply) error {
```

Comment:
About HandleSignRequest + HandleSignReply
* No need for them to be exported. handleSign* is enough I think.
* A small comment would be most welcomed. Same for handleSignReply.